app/src/androidTest/kotlin/at/bitfire/davdroid/sync/SyncAdapterServicesTest.kt

```diff
@@ -87,6 +85,10 @@ class SyncAdapterServicesTest {
         TestUtils.setUpWorkManager(context, workerFactory)
 
         account = TestAccount.create()
+
+        ContentResolver.setMasterSyncAutomatically(true)
```

Comment:
As in `AndroidSyncFrameworkTest`, we should not change global system state (= restore the automatic sync after the tests).